pages/Subpages/[detailCountry].js

```diff
@@ -55,17 +57,26 @@ export default function DetailCountryPage({
   function closeButton() {
     newImageDialog.close();
   }
-  function newImgDialog() {
+  function addNewImgDialog() {
     newImageDialog.showModal();
   }
 
-  if (currentCountry === undefined) {
-    return null;
+  if (!currentCountry || currentCountry.length === 0) {
+    return (
+      <>
+        <ErrorMessage>Error: Es ist kein Bild vorhanden</ErrorMessage>
+        <Link href={"/"}>Startseite</Link>{" "}
+      </>
+    );
   }
 
   function getImages(data) {
-    setGetImagesData([...getImagesData, data]);
-    setIsSaved(true);
+    if (data) {
+      return setGetImagesData([...getImagesData, data]), setIsSaved(true);
```

Comment:
Somewhat confusing to read. For improved readability I'd recommend this instead:
```suggestion
      setGetImagesData([...getImagesData, data]);
      setIsSaved(true);
      return;
```
It's mostly the same thing, not quite, but it's more accessible to the eye.